var/spack/repos/builtin/packages/libpeas/package.py

```diff
@@ -86,7 +86,7 @@ def setup_run_environment(self, env):
 
     def autoreconf(self, spec, prefix):
         autoreconf_args = ['-ivf']
-        aclocal_pkg_list = ['pkg-config',
+        aclocal_pkg_list = ['pkgconf',
```

Comment:
Since we depend on the virtual spec, it's probably better to use that:
```suggestion
        aclocal_pkg_list = ['pkgconfig',
```

Reply:
I modify it to pkgconfig.